### PR TITLE
Issue #10421 fix jpms warning

### LINKED ISSUE #10421: WARNING: Unknown module
**Jetty version(s)**
12

**Description**

In a clean successful build (e.g. [1482](https://jenkins.webtide.net/blue/rest/organizations/jenkins/pipelines/jetty.project/branches/jetty-12.0.x/runs/1482/nodes/7/steps/33/log/?start=0) there are at least 15 warnings like:
```
WARNING: Unknown module: org.eclipse.jetty.http2.server specified to --add-reads
```

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

**Comment by joakime** [comment]
@olamy The Eclipse ECA thinks this commit belongs to your gmail address, not your apache address.

You might need to redo this entirely from scratch.

Tip: if you have an ECA issue, don't cherry-pick, as the Eclipse ECA tool remembers past commits, even on different branches / PRs.  (This commit is also on PR #10423 and that PR has your gmail address)

**Comment by olamy** [comment]
sounds like a bug in Eclipse tooling...
```
git log

commit 23845d0040d8c58c7c00eb9e7c5ecaf5cac23a26 (HEAD -> jetty-12.0.x-fix-jpms-warning, origin/jetty-12.0.x-fix-jpms-warning)
Author: Olivier Lamy <olamy@apache.org>
Date:   Mon Aug 28 20:45:04 2023 +1000

    remove wrong jpms declaration
    
    Signed-off-by: Olivier Lamy <olamy@apache.org>

``` 
or 
```
olamy@pop-os:~/dev/sources/jetty/jetty.project$ git show 23845d0040d8c58c7c00eb9e7c5ecaf5cac23a26
commit 23845d0040d8c58c7c00eb9e7c5ecaf5cac23a26 (HEAD -> jetty-12.0.x-fix-jpms-warning, origin/jetty-12.0.x-fix-jpms-warning)
Author: Olivier Lamy <olamy@apache.org>
Date:   Mon Aug 28 20:45:04 2023 +1000

    remove wrong jpms declaration
    
    Signed-off-by: Olivier Lamy <olamy@apache.org>

```
even the commit is marked as verified by my key...

Screenshot from 2023-08-29 08-46-07


https://keyserver.ubuntu.com/pks/lookup?search=873A8E86B4372146&fingerprint=on&op=index

not sure what else I can do to prove it's really me :)

**Comment by olamy** [comment]
@joakime no need to redo code etc..
just a rebase  and mark this commit as editable then 
```
git commit --amend --author="Olivier Lamy <olamy@apache.org>" --no-edit
git push --force
```

really sounds like a bug in the Eclipse tooling

File: jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod [hunk 1]
```diff
@@ -19,9 +19,4 @@ lib/ee8-websocket/jetty-ee8-websocket-servlet-${jetty.version}.jar
 lib/ee8-websocket/jetty-javax-websocket-api-@jakarta.websocket.api.version@.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-client-${jetty.version}.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-common-${jetty.version}.jar
-lib/ee8-websocket/jetty-ee8-websocket-javax-server-${jetty.version}.jar
-
-[jpms]
-# The implementation needs to access method handles in
-# classes that are in the web application classloader.
-add-reads: org.eclipse.jetty.websocket.javax.common=ALL-UNNAMED
+lib/ee8-websocket/jetty-ee8-websocket-javax-server-${jetty.version}.jar
```
Comment (janbartel) [inline jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod:22]: If this line was missing, does this mean that we don't have a distribution test for ee8 javax websocket?

**Comment by janbartel** [comment]
@olamy so many `[jpms]` clauses removed for websocket - are they truly not needed?

File: jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod (hunk 1 above)
Comment (olamy) [reply jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod:22]: > If this line was missing, does this mean that we don't have a distribution test for ee8 javax websocket?

which line was missing? I only removed the `jpms` lines here

**Comment by gregw** [comment]
> @olamy so many `[jpms]` clauses removed for websocket - are they truly not needed?

@lachlan-roberts ?

**Comment by olamy** [comment]
@janbartel they 

> @olamy so many `[jpms]` clauses removed for websocket - are they truly not needed?


There are only 3 module names removed and none of them is referenced in any of the module-info.java nor in any pom.xml of the project.

**Review by janbartel — APPROVED** [review]

**Pull request merged**